pkg/vmprovider/providers/vsphere/virtualmachine/configspec.go

```diff
@@ -90,9 +96,10 @@ func CreateConfigSpecForPlacement(
 	vmClassSpec *v1alpha1.VirtualMachineClassSpec,
 	minFreq uint64,
 	storageClassesToIDs map[string]string,
+	imageFirmware string,
 	vmClassConfigSpec *vimtypes.VirtualMachineConfigSpec) *vimtypes.VirtualMachineConfigSpec {
 
-	configSpec := CreateConfigSpec(vmCtx.VM.Name, vmClassSpec, minFreq, vmClassConfigSpec)
+	configSpec := CreateConfigSpec(vmCtx.VM.Name, vmClassSpec, minFreq, imageFirmware, vmClassConfigSpec)
```

Comment:
TODO: make the function take `CreateConfigSpecOpts` or something like that. Not right now, just to simplify it later.

Reply:
will mark a TODO here. Thanks!